include/spdlog/details/spdlog_impl.h

```diff
@@ -61,7 +62,11 @@ inline std::shared_ptr<spdlog::logger> spdlog::daily_logger_st(const std::string
 inline std::shared_ptr<spdlog::logger> create_console_logger(const std::string& logger_name, spdlog::sink_ptr sink, bool color)
 {
     if (color) //use color wrapper sink
+#ifdef _WIN32
+        sink = std::make_shared<spdlog::sinks::wincolor_sink>(sink);

```

Comment:
Can you use ANSI sequences on Windows automatically if `ANSICON` environment variable is set, because windows console not support all features? Programs like ConEmu or AnsiCon set this variable when they support ANSI escape codes (http://conemu.github.io/en/AnsiEscapeCodes.html).